### PR TITLE
[Typing] 修改一些示例代码中的类型错误 part 2

### PR DESCRIPTION
**Pull request by megemini**
### PR Category

Others

### PR Types

Others

### Description

关联 PR https://github.com/PaddlePaddle/Paddle/issues/65008 https://github.com/PaddlePaddle/Paddle/pull/65397

@SigureMo

### CODE REVIEW

**Review by SigureMo — DISMISSED** [review]
LGTMeow

File: python/paddle/incubate/operators/graph_khop_sampler.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def graph_khop_sampler(
             >>> colptr = paddle.to_tensor(colptr, dtype="int64")
             >>> nodes = paddle.to_tensor(nodes, dtype="int64")
 
-            >>> edge_src, edge_dst, sample_index, reindex_nodes = paddle.incubate.graph_khop_sampler(row, colptr, nodes, sample_sizes, False)
+            >>> edge_src, edge_dst, sample_index, reindex_nodes = paddle.incubate.graph_khop_sampler.graph_khop_sampler(row, colptr, nodes, sample_sizes, False)
```
Comment (SigureMo) [inline python/paddle/incubate/operators/graph_khop_sampler.py]: image

File: python/paddle/incubate/autograd/primapi.py
Comment (SigureMo) [inline python/paddle/incubate/autograd/primapi.py:1]: image

示例代码好像有点问题

File: python/paddle/incubate/operators/graph_khop_sampler.py (hunk 1 above)
Comment (megemini) [reply python/paddle/incubate/operators/graph_khop_sampler.py]: 这段示例代码跑不了 ～ 🫠 我把它复原吧 ～

File: python/paddle/incubate/autograd/primapi.py
Comment (megemini) [reply python/paddle/incubate/autograd/primapi.py:1]: 这段代码也有问题 ～ 造孽啊 ... ...

**Comment by megemini** [comment]
### Update 20240626

- 代码本身存在问题，退回 [python/paddle/incubate/autograd/primapi.py](https://github.com/PaddlePaddle/Paddle/pull/65463/files/6a24072e4b25d1fac201955da27a79e71d20521e#diff-e8130e40a9b271b9e558cbd057b1e4aba7258900f37160cf39ed4fcc1b38c871) 和 [python/paddle/incubate/operators/graph_khop_sampler.py](https://github.com/PaddlePaddle/Paddle/pull/65463/files/6a24072e4b25d1fac201955da27a79e71d20521e#diff-387c38b819c369f7956ab50ffade6900628f1efa78f97a2265e3d35a6bf8549b) 的示例修改 ～

**Review by SigureMo — APPROVED** [review]

**Pull request merged**